### ISSUE TITLE
Use descriptive alt-text

### ISSUE THREAD

**Issue by ciaoaaron** [Thurst5679/PAD_website#3]
https://github.com/Thurst5679/PAD_website/blob/main/index.html#L32

Good job adding the alt text. It should be even better if the alt text is more descriptive. Please refer to S6's material.
